packages/lab/src/ProgressBar/ProgressBar.js

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import PropTypes from "prop-types";
+import clsx from "clsx";
+
+import clamp from "lodash/clamp";
+import { HvTypography } from "@hitachivantara/uikit-react-core";
+
+import { withStyles } from "@material-ui/core";
+
+import styles from "./styles";
+
+/**
+ * ProgressBar provides feedback about a process that is taking place in the application.
+ */
+const HvProgressBar = (props) => {
+  const {
+    className,
+    classes,
+    value = 0,
+    status,
+    undeterminate = false,
+    ariaProps,
+    ...others
+  } = props;
+
+  const clampedValue = clamp(value, 0, 100);
+
+  return (
+    <div
+      className={clsx(className, classes.root, classes.progress)}
+      role="progressbar"
+      status="inProgress"
+      aria-valuenow={clampedValue} // it only makes sense to put this in ariaProps if the min and max values of the progress bar are intended to be changed by the user
+      aria-valuemin="0"
+      aria-valuemax="100"
+      {...others}
+    >
+      <div className={classes.progressContainer}>
+        <div
```

Comment:
ariaProps -> labelProps
see if we can eliminate the div and use only the `HvTypography`